numba/core/interpreter.py

```diff
@@ -1031,6 +1303,7 @@ def interpret(self, bytecode):
             # any large number of arguments from a list to a
             # tuple.
             peepholes.append(peep_hole_call_function_ex_to_call_function_kw)
+            peepholes.append(peep_hole_fuse_dict_add_updates)
```

Comment:
No. If run before `peep_hole_call_function_ex_to_call_function_kw` you can delete `_call_function_ex_replace_kws_large` because the dictionary used in varkwargs will always be a single build_map that is fully populated.

Reply:
I also think this change is worth making to reduce code complexity.

Reply:
This is now just no.

Reply:
> This is now just no.

Thanks for confirming!